test/test_data.py

```diff
@@ -6,7 +6,7 @@
 def test_adult_col():
     data_name = "adult"
     data_catalog = "adult_catalog.yaml"
-    data_catalog = DataCatalog(data_name, data_catalog)
+    data_catalog = DataCatalog(data_name, data_catalog, True)
```

Comment:
Passing just a `True` is always a bit tricky since its hard to infer what that referes to. Can you use the named parameter here?

Reply:
👍 Makes perfectly sense, will change it in the next commit